##### Relativité et quantique obsolescence programmée...

Einstein en interprétant falacieusement le coefficient de Lorentz, a dirigé la physique dans un gouffre infini.

Le résultat est la célérité du photon (on voit en un siècle la transformation de vitesse de la lumière en célérité du photon).

Donc la question est : Quelle est la cause de cette célérité ? 

La physique l'a déterminé à partir du rapport $\frac{Qe}{Qm}$ comme dimension d'une vitesse. Je dirais plutôt une impédance !

In [2]:
%display latex
Qe=299792458^2;Qm=299792458;Qe/Qm

299792458

$Qe$ correspond au coefficient de Coulomb moins 7 décimales : $\pi *4E-7$ par rapport au $c^2$

In [38]:
Cc=1/(4*n(pi)*8.85418782E-12);Cc#  7 décimales - en Newtons

8.98755178495272e9

In [41]:
Qe # 17 décimales en Km

89875517873681764

In [11]:
89875517873681764.0/8987551784952723.0

10.00000000268755

In [29]:
c=299792458;mu=n(pi,digits=35)*4E-7;ep=1/(mu*c*c)
cc=1/(mu*ep);(cc)

8.98755178736818e16

In [42]:
Cc=1/(4*n(pi)*ep);Cc#  7 décimales -6 pour kg

8.98755178736818e9

In [43]:
int(cc/Cc)

10000000

On voit que Coulomb a mesuré une particule $kc$, inférieure de un dix millionième du photon. C'était déjà une prouesse que de la mesurer, aujourd'hui ce ne serait pas évident de le faire avec tout la pollution, le bruit et les tremblements !

Le coefficient de Coulomb est une particule statique puisque c'est un carré (pair/impair) dont on ne peut extraire aucune racine carrée parfaite, pour cela il faudrait que le nombre de décimale soit pair, j'en tire un autre renseignement : la particule lumineuse (photon) est une particule bipolaire, sinon on obtiendrait pas le même résultat de la racine carrée : D'où 2 racines, une paire et une impaire :

Kc : Racine carrée du coeffient de Colomb  $\sqrt(kc)= 94802.6

cc= Racine carrée $\sqrt(c^2)=299792458$

In [30]:
sqrt(Cc)

94802.6992620367

In [33]:
sqrt(cc)

2.99792458000000e8

In [37]:
sqrt(89875.51787368177) # méga mètre

299.792458000000

En changeant la parité de $kc$, en lui mettant un résulat paire, on obtient la célérité dans une autre unité : $\sqrt(89875.51787368177)=299.792458$ Méga mètres